imagetool/src/main/java/com/oracle/weblogic/imagetool/installer/FmwInstallerType.java

```diff
@@ -80,7 +80,9 @@ public enum FmwInstallerType {
     WCS(Utils.toSet(FMW.products, AruProduct.WCS),
         InstallerType.FMW, InstallerType.WCS),
     OHS(Collections.singleton(AruProduct.OHS),
-        InstallerType.OHS)
+        InstallerType.OHS),
+    ODI(Collections.singleton(AruProduct.ODI),
```

Comment:
According to the documentation at https://docs.oracle.com/cd/E17904_01/install.1111/e16453/install.htm#ODING285, a WebLogic Server install is required for ODI.  But, you do not have WLS or FMW as part of the installer list here.

Reply:
No, I think ODI is the correct name.  If you add ODI on WLS, we can call that one ODI_WLS.  This one should stay ODI.

Reply:
Ok. We will retain the name as it is.